pycipapi/cipapi_client.py

```diff
@@ -126,6 +127,14 @@ def submit_clinical_report_raw(self, payload, partner_id, analysis_type, report_
         url = self.build_url(self.url_base, self.CR_ENDPOINT, partner_id, analysis_type, report_id) + '/'
         return self.post(url, payload, params=params)
 
+    def submit_variant_interpretation_logs_raw(self, payload, case_id, case_version, **params):
```

Comment:
_logs_ or _log_ ?

e.g. if its a single log for a single case then maybe "log" is better than "logs" ?

Reply:
It's for a single case, but you can post a list of logs (one for each variant)